shared/src/jvmMain/kotlin/org/jetbrains/kotlinconf/utils/Dispatchers.jvm.kt

```diff
@@ -0,0 +1,7 @@
+package org.jetbrains.kotlinconf.utils
+
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+
+actual val Dispatchers.App: CoroutineDispatcher
```

Comment:
As I understand we may use Dispatchers.Main here

Reply:
The `Dispatchers.Main` purpose is UI updates. This dispatcher is used for IO operations